<!--
Setup real robot of Mikata Arm type 2.
Using mikata_driver = controller as ROS node.
-->

<launch>

  <arg name="jsdev" default="/dev/input/js0" />
  <include file="$(find ay_util)/launch/mikata/mikata_cmn.launch">
    <arg name="jsdev"  value="$(arg jsdev)" />
  </include>

  <arg name="dev" default="/dev/ttyUSB0" />
  <node pkg="ay_util" type="mikata_driver.py"
        name="mikata_driver" output="screen"
        args="$(arg dev) Mikata" />

  <!--<node pkg="rviz" type="rviz" name="rviz"/>-->

</launch>
